docs/en/sql-reference/functions/tuple-map-functions.md

```diff
@@ -429,6 +429,38 @@ Result:
 │ {}                         │
 └────────────────────────────┘
 ```  
+
+
+## mapFromArrays {#mapFromArrays}  
```

Comment:
Function `mapFromArrays()` constructs a map. We should list it's documentation after the docs for function `map()` (l. 68) which is the other described function in this file which constructs a map.